CONTRIBUTING.md

```diff
@@ -26,7 +26,7 @@ for using [MkDocs](https://www.mkdocs.org):
    directory by running `python -m venv .venv`. Each time you open a new
    command window, activate the environment before running `mkdocs` commands:
     - Windows users: run `.venv\Scripts\activate`
-    - Linux or Mac users: run `source .venv/Scripts/activate`
+    - Linux or Mac users: run `source .venv/bin/activate`
```

Comment:
On Debian/Ubuntu the path is bin instead of Scripts. @prestwick could you please confirm this is correct for Mac?

Reply:
Confirmed